scratch/evaluate_qd_channel_lroom_scenario.cc

```diff
@@ -253,6 +253,8 @@ main (int argc, char *argv[])
   double simulationTime = 10;                   /* Simulation time in seconds. */
   bool pcapTracing = false;                     /* PCAP Tracing is enabled or not. */
   std::map<std::string, std::string> tcpVariants; /* List of the tcp Variants */
+  uint16_t tos = 0;                             /* Select AC_BE as default AC */
```

Comment:
tos should be uint8_t looking at InetSocketAddress::SetTos

Reply:
It's just an index to tosValues vector. I changed the name of this variable in the last commit. It should be more clear now.